bcda/router.go

```diff
@@ -32,6 +32,7 @@ func NewRouter() http.Handler {
 		if os.Getenv("DEBUG") == "true" {
```

Comment:
Is this only supposed to be enabled in debug mode?

Reply:
Is it?? I'm not too sure either way.

Reply:
It should be accessible in all environments.

Reply:
Will do